dialogs/graph_settings.py

```diff
@@ -67,8 +67,23 @@ def __init__(self, parent):
         # Connect and show
         self.OKButton.clicked.connect(self.accept_settings)
         self.cancelButton.clicked.connect(self.close)
-
+        self.radio_range_manual.clicked.connect(lambda: self.toggle_manual(True))
+        self.radio_range_auto.clicked.connect(lambda: self.toggle_manual(False))
+        
+        if self.radio_range_auto.isChecked():
+           self.toggle_manual(False)
+           
+        elif self.radio_range_manual.isChecked():
+            self.toggle_manual(True)
+ 
         self.exec_()
+    
+    def toggle_manual(self, isManual):
+        isDisabled = not isManual #if manual is on, spinboxes are enabled
+        self.spin_range_x_min.setDisabled(isDisabled)
+        self.spin_range_x_max.setDisabled(isDisabled)
+        self.spin_range_y_min.setDisabled(isDisabled)
+        self.spin_range_y_max.setDisabled(isDisabled) 
```

Comment:
```suggestion
    def toggle_manual(self, is_manual):
        # if manual is on, spinboxes are enabled
        is_disabled = not is_manual
        self.spin_range_x_min.setDisabled(is_disabled )
        self.spin_range_x_max.setDisabled(is_disabled )
        self.spin_range_y_min.setDisabled(is_disabled )
        self.spin_range_y_max.setDisabled(is_disabled ) 
```
Code style issues: 
- variables should be snakecase
- at least two spaces before inline comment
- inline comment should start with '# '